component/Grids/index.html

```diff
@@ -79,7 +81,7 @@ <h4 class="component-subheading"> Grid with two column </h4>
 
         <!-- GRID 3 COLUMN -->
 
-        <h4 class="component-subheading"> Grid with three column </h4>
+        <h4 class="component-subheading">> Grid with three column </h4>
```

Comment:
Remove extra closing bracket ">"

Reply:
Its just for showing sub points.
![image](https://user-images.githubusercontent.com/63742296/154865921-a6854abb-724d-41e5-a9d3-049699f44bee.png)